src/libstd/sys/unix/fs.rs

```diff
@@ -149,32 +155,51 @@ impl FileAttr {
 #[cfg(not(any(target_os = "ios", target_os = "macos", target_os = "netbsd")))]
 impl FileAttr {
     pub fn modified(&self) -> io::Result<SystemTime> {
-        Ok(SystemTime::from(libc::timespec {
-            tv_sec: self.stat.st_mtime as libc::time_t,
-            tv_nsec: self.stat.st_mtime_nsec as libc::c_long,
-        }))
+        #[cfg(any(target_env = "musl", target_os = "emscripten"))]
+        { return Ok(SystemTime::from( self.stat.st_mtim )); }
+        #[cfg(not(any(target_env = "musl", target_os = "emscripten")))]
+        {
+            return Ok(SystemTime::from(libc::timespec {
+                tv_sec: self.stat.st_mtime as libc::time_t,
+                tv_nsec: self.stat.st_mtime_nsec as libc::c_long,
+            }));
+        }
     }
 
     pub fn accessed(&self) -> io::Result<SystemTime> {
-        Ok(SystemTime::from(libc::timespec {
-            tv_sec: self.stat.st_atime as libc::time_t,
-            tv_nsec: self.stat.st_atime_nsec as libc::c_long,
-        }))
+        #[cfg(any(target_env = "musl", target_os = "emscripten"))]
```

Comment:
Some of these additions of `musl` may not be necessary because the standard library is already guaranteed to compile for `x86_64-unknown-linux-musl`